code/datums/gamemode/role/role.dm

```diff
@@ -264,11 +267,11 @@
 		else
 			to_chat(antag.current, "<img src='data:image/png;base64,[icon2base64(logo)]' style='position: relative; top: 10;'/> <B>You are \a [name][faction ? ", a member of the [faction.GetObjectivesMenuHeader()]":"."]</B>")
 
-/datum/role/proc/PreMindTransfer(var/datum/mind/M)
-	return
+/datum/role/proc/PreMindTransfer(var/mob/living/old_character)
+	return TRUE
```

Comment:
i really like useless confusing return values

Reply:
I usually put a `return TRUE` in an empty proc, in case it's evaluated as a boolean.
But if you really want it I can change it. I don't think it's that big of a deal, especially since what the procs do is explained at the top of the file.

Reply:
I don't think being able to chain things using && is a good reason to pass useless TRUEs around. I'd much rather have a clear distinction between what's supposed to return something and what's not. But that's just me. Feel free to leave it as is, it's just my take on it.

Reply:
I don't care much either way, but it's a fair point. I'll change it.